### DISCUSSION TITLE
Software update

### DISCUSSION THREAD

**Discussion by Scofis** [linux-surface/linux-surface#1234] [Q&A] [answered]
Now that I have the kernal installed, how should I handle updates?

I see "kernal" brought up sometimes in the Software Updater for Ubuntu. And I'm kinda worried that it will undo (somehow) the Surface-kernal. Or is that not a thing.

**Comment by qzed** [comment, marked as answer]
You can update completely normally. Different kernel flavors are installed in parallel to each other. So the generic Ubuntu kernel is installed and updated in parallel to our Surface kernel. The only thing that could happen is that your bootloader (e.g. grub) decides to boot into a different kernel by default after the update. You can check that with `uname -a` and, if necessary, specifically select the Surface kernel in the bootloader options (or configure your bootloader accordingly).

**Reply by Scofis** [reply]
Thank you so much. That is good to know.

So the updater will automatically update the Surface-kernal?

**Reply by qzed** [reply]
Yes, as long as you've followed the install guide and set up the repos (and not downloaded the package files manually) all the pack packages you've installed from that repo will get updated as well. Including the kernel.

**Reply by Scofis** [reply]
That answers my question. Thank you for your time!
